src/fable-publish-utils/PublishUtils.fs

```diff
@@ -584,7 +584,21 @@ module Publish =
                     |> function
                         | Some x -> x
                         | None -> failwithf "Cannot find .nupgk for %s" projDir
+
+                let snupkg =
+                    IO.Directory.GetFiles(tempDir)
+                    |> Seq.tryPick (fun path ->
+                        if path.EndsWith(".snupkg")
+                        then Some(tempDir </> path)
+                        else None)
+                    |> function
+                        | Some x -> x
+                        | None -> failwithf "Cannot find .snupgk for %s" projDir
+
                 runList ["dotnet nuget push"; nupkg; "-s nuget.org -k"; nugetKey]
+
+                // Publish snupkg after the nupkg, since you need to already have a corresponding nupkg to match the symbols against
+                runList ["dotnet nuget push"; snupkg; "-s nuget.org -k"; nugetKey]
```

Comment:
@cartermp I got this when trying to push a package so it seems the first `dotnet nuget push` command is already detecting and pushing the symbols (which makes the second command fail because they're already published) so I have commented out this line.

```
> dotnet nuget push src/Fable.AST/temp/Fable.AST.3.1.1.nupkg -s nuget.org -k ...
Pushing Fable.AST.3.1.1.nupkg to 'https://www.nuget.org/api/v2/package'...
  PUT https://www.nuget.org/api/v2/package/
  Created https://www.nuget.org/api/v2/package/ 1690ms
Your package was pushed.
Pushing Fable.AST.3.1.1.snupkg to 'https://www.nuget.org/api/v2/symbolpackage'...
  PUT https://www.nuget.org/api/v2/symbolpackage/
  Created https://www.nuget.org/api/v2/symbolpackage/ 725ms
Your package was pushed.
> dotnet nuget push src/Fable.AST/temp/Fable.AST.3.1.1.snupkg -s nuget.org -k ...
Pushing Fable.AST.3.1.1.snupkg to 'https://www.nuget.org/api/v2/symbolpackage'...
  PUT https://www.nuget.org/api/v2/symbolpackage/
  Conflict https://www.nuget.org/api/v2/symbolpackage/ 843ms
To skip already published packages, use the option --skip-duplicate
```

Reply:
Makes sense! Interesting that it all just works with the snupkg.

Reply:
If we're not sure what are the conditions to trigger the symbol publishing maybe it's better to still issue the second command with `-skip-duplicate` just to be sure 🤔